### PR TITLE
fix: peckShield audit PVE-004 (Sanity Checks)

### PR DESCRIPTION
**Pull request by UrAvgDeveloper**
## Description

Fixes PVE-004

## Checklist

- [x] I have updated the documentation to account for the changes in the code.
- [x] If I added new functionality, I added tests covering it.
- [x] If I fixed a bug, I added a test preventing this bug from silently reappearing again.
- [x] My contribution follows [Venus contribution guidelines](docs/CONTRIBUTING.md).

### CODE REVIEW

**Comment by defcon022** [comment]
can we have a test case for the same fix?

File: tests/hardhat/Comptroller/accountLiquidityTest.ts [hunk 1]
```diff
@@ -298,6 +298,53 @@ describe("Comptroller", () => {
...
+        supportMarket: true,
+        collateralFactor: cf2,
+        underlyingPrice: up2,
+        poolRegistry,
+      });
+      await comptroller.connect(user).enterMarkets([vToken1.address, vToken2.address]);
+      //await quickMint(vToken1, user, amount1);
+      vToken1.getAccountSnapshot.whenCalledWith(userAddress).returns([0, amount1, 0, convertToUnit("1", 18)]);
+      //await quickMint(vToken2, user, amount2);
+      vToken2.getAccountSnapshot.whenCalledWith(userAddress).returns([0, amount2, 0, convertToUnit("1", 18)]);
```
Comment (kkirka) [inline tests/hardhat/Comptroller/accountLiquidityTest.ts:334]:
```suggestion
      // pretend user mints amount1 of vToken1
      vToken1.getAccountSnapshot.whenCalledWith(userAddress).returns([0, amount1, 0, convertToUnit("1", 18)]);
      // pretend user mints amount2 of vToken2
      vToken2.getAccountSnapshot.whenCalledWith(userAddress).returns([0, amount2, 0, convertToUnit("1", 18)]);
```

File: tests/hardhat/Comptroller/accountLiquidityTest.ts [hunk 2]
```diff
@@ -298,6 +298,53 @@ describe("Comptroller", () => {
...
+      const dummyAddr2 = await accounts[2].getAddress();
+      const param = {
+        vTokenCollateral: dummyAddr1,
+        vTokenBorrowed: dummyAddr2,
+        repayAmount: 1e3,
+      };
+      await expect(comptroller.liquidateAccount(userAddress, [param])).to.be.revertedWithCustomError(
+        comptroller,
+        "MarketNotListed",
+      );
```
Comment (kkirka) [inline tests/hardhat/Comptroller/accountLiquidityTest.ts:346]:
```suggestion
      ).withArgs(dummyAddr1);
```

File: tests/hardhat/Comptroller/accountLiquidityTest.ts [hunk 3]
```diff
@@ -298,6 +298,53 @@ describe("Comptroller", () => {
...
+        poolRegistry,
+      });
+      await comptroller.connect(user).enterMarkets([vToken1.address, vToken2.address]);
+      //await quickMint(vToken1, user, amount1);
+      vToken1.getAccountSnapshot.whenCalledWith(userAddress).returns([0, amount1, 0, convertToUnit("1", 18)]);
+      //await quickMint(vToken2, user, amount2);
+      vToken2.getAccountSnapshot.whenCalledWith(userAddress).returns([0, amount2, 0, convertToUnit("1", 18)]);
+      await comptroller.setMinLiquidatableCollateral(3002719);
+      const dummyAddr1 = await accounts[1].getAddress();
+      const dummyAddr2 = await accounts[2].getAddress();
```
Comment (kkirka) [inline tests/hardhat/Comptroller/accountLiquidityTest.ts:337]: hardhat-ethers' `getSigner()` returns `SignerWithAddress`, so you can use just `accounts[*].address` here

**Review by kkirka — APPROVED** [review]

**Review by defcon022 — APPROVED** [review]

**Pull request merged**